fendermint/vm/interpreter/Cargo.toml

```diff
@@ -32,11 +32,19 @@ fvm_ipld_blockstore = { workspace = true }
 fvm_ipld_encoding = { workspace = true }
 fvm_ipld_car = { workspace = true }
 
-[dev-dependencies]
-quickcheck = { workspace = true }
+futures-core = "0.3.28"
+futures-util = "0.3.28"
+libipld = { version = "0.14", default-features = false, features = ["dag-cbor"] }
 tokio = { workspace = true }
+tokio-stream = "0.1.14"
+tokio-util = {version = "0.7.8", features = ["compat"]}
+pin-project = "1.1.2"
```

Comment:
I would move these to the top level Cargo.toml file, even if nothing else uses them at the moment. We have a lot of crates already and it might be impossible to spot common ones later.